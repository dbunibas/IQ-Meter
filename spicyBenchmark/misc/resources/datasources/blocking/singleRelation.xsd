<?xml version="1.0" encoding="UTF-8"?>
<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema">
   <xs:element name="e1">
      <xs:complexType>
         <xs:sequence>
            <xs:element name="Persons" nillable="false" minOccurs="1">
               <xs:complexType>
                  <xs:sequence>
                     <xs:element maxOccurs="unbounded" name="Person" nillable="false" minOccurs="1">
                        <xs:complexType>
                           <xs:sequence>
                              <xs:element name="pid" type="xs:string" nillable="false" minOccurs="1"/>
                              <xs:element name="tname" type="xs:string" nillable="false" minOccurs="1"/>
                              <xs:element name="resource" type="xs:string" nillable="false" minOccurs="1"/>
                              <xs:element name="aattribute" type="xs:string" nillable="false" minOccurs="1"/>
                              <xs:element name="battribute" type="xs:string" nillable="false" minOccurs="1"/>
                              <xs:element name="cattribute" type="xs:string" nillable="false" minOccurs="1"/>
                              <xs:element name="dattribute" type="xs:string" nillable="false" minOccurs="1"/>
                              <xs:element name="eattribute" type="xs:string" nillable="false" minOccurs="1"/>
                           </xs:sequence>
                        </xs:complexType>
                     </xs:element>
                  </xs:sequence>
               </xs:complexType>
            </xs:element>
         </xs:sequence>
      </xs:complexType>
      <xs:key name="keyPid">
            <xs:selector xpath=".//Person"/>
            <xs:field xpath="pid"/>
        </xs:key>
   </xs:element>
</xs:schema>