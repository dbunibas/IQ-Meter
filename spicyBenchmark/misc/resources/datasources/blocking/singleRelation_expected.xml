<?xml version="1.0" encoding="US-ASCII"?>
<e1>
	<Persons>
		<Person>
			<pid>c1</pid>
			<tname>c2</tname>
			<resource>_L3</resource>
		</Person>	
		<Person>
			<pid>c3</pid>
			<tname>c4</tname>
			<resource>c5</resource>
		</Person>
		<Person>
			<pid>c3</pid>
			<tname>c4</tname>
			<resource>_L9</resource>
		</Person>
		<Person>
			<pid>c6</pid>
			<tname>c7</tname>
			<resource>c8</resource>
		</Person>
		<Person>
			<pid>c1</pid>
			<tname>c1</tname>
			<resource>c1</resource>
		</Person>
		<Person>
			<pid>A</pid>
			<tname>B</tname>
			<resource>C</resource>
		</Person>
		<Person>
			<pid>D</pid>
			<tname>E</tname>
			<resource>F</resource>
		</Person>
		<Person>
			<pid>A</pid>
			<tname>B</tname>
			<resource>_L0</resource>
		</Person>
	</Persons>
</e1>